specs/ui-popover/popover-trigger.directive.spec.ts

```diff
@@ -70,15 +72,15 @@ describe(`popover trigger`, () => {
 
     describe(`ID`, () => {
       test(`should support a custom ID`, fakeAsync(() => {
-        setup(Basic);
+        setup(testComponents.Basic);
```

Comment:
Is there a reason to go back to this format? If we just use `Basic` the IDE will handle the import for us

Reply:
ah no, I think I used one example (input probably) and it might just have out of dated format..